### DISCUSSION TITLE
logica for json

### DISCUSSION THREAD

**Discussion by wdduncan** [EvgSkv/logica#299] [Ideas]
Just a thought: Would it be feasible to take the same approach to create a logica framework for working with json (or yaml) content? E.g.: produce [jq](https://jqlang.github.io/jq/manual/) statements for querying.

Again ... this just a thought I had. 

I really like what you have done with logica!

**Comment by EvgSkv** [comment]
Thank you @wdduncan !

SQLite is quite good with JSON. Logica can leverage that to process JSON: [EXAMPLE](https://colab.research.google.com/drive/12mwBKGwZs74SkHI3ebD8showu90VIW8V?usp=sharing).

Supporting non-SQL engines in Logica is work, but should be feasible if needed.

**Reply by wdduncan** [reply]
Thanks for the JSON example!
